### DISCUSSION TITLE
Forcing a journal entry on edit?

### DISCUSSION THREAD

**Discussion by Redxd** [netbox-community/netbox#14835] [Q&A]
Is there a way to force a user to enter in a journal entry everytime an edit is about to be commited to?

**Comment by jeremystretch** [comment]
No; this is the purpose of the [change log](https://docs.netbox.dev/en/stable/features/change-logging/), which automatically records any changes. Journal entries are there as a means to record arbitrary human-relevant notes.

**Reply by Redxd** [reply]
But won't that data dissapear after the 90days change log retention? Would the journal entries not stick with the device?

**Reply by jeremystretch** [reply]
You can configure the [changelog retention time](https://docs.netbox.dev/en/stable/configuration/miscellaneous/#changelog_retention), even disabling it entirely if you'd like.
